module/models/item-archetype.mjs

```diff
@@ -22,6 +22,7 @@ export default class CrucibleArchetypeItem extends foundry.abstract.TypeDataMode
       }, {}), {validate: CrucibleArchetypeItem.#validateAbilities}),
       talents: new fields.SetField(new fields.DocumentUUIDField({type: "Item"})),
       skills: new fields.SetField(new fields.StringField({required: true, choices: SYSTEM.SKILLS})),
+      equipment: new fields.SetField(new fields.DocumentUUIDField({type: "Item"})),
```

Comment:
I think we need a bit more than this because we need to support:
- Quantity (consider an assassin that should have 2x daggers)
- Equipped state (perhaps need to record whether the items should be initially equipped or not)

I think maybe something like:
```suggestion
      equipment: new fields.ArrayField(new fields.SchemaField({
        item: new fields.DocumentUUIDField({type: "Item"}),
        quantity: new fields.NumberField({required: true, nullable: false, integer: true, initial: 1}),
        equipped: new fields.BooleanField()
      })),
```
This would give us a bit of room to grow, if we need to configure any other properties about granted equipment in the future.